QEfficient/cloud/execute.py

```diff
@@ -30,15 +31,24 @@ def main(
     :param model_name: str. Hugging Face Model Card name, Example: [gpt2]
     :prompt: str. Sample prompt for the model text generation
     :qpc_path: str.  Path to the generated binary after compilation.
-    :devices: List[int]. Device Ids to be used for compilation. if devices > 1. Multiple Card setup is enabled.
+    :device_group: List[int]. Device Ids to be used for compilation. if len(device_group) > 1. Multiple Card setup is enabled.
```

Comment:
Add description for `prompts_txt_file_path`

Reply:
done